src/main/java/net/geforcemods/securitycraft/entity/SentryEntity.java

```diff
@@ -165,8 +168,18 @@ else if(item == SCContent.DISGUISE_MODULE.get())
 				ItemStack module = getDisguiseModule();
 
 				if(!module.isEmpty()) //drop the old module as to not override it with the new one
+				{
 					Block.spawnAsEntity(world, getPosition(), module);

```

Comment:
You are saving `getPosition()` in a variable. If you create such a variable, please also use it in its scope wherever possible.

Reply:
`getPosition` is also used in other places in `processInteract` than only in the disguise module part. Should I replace it there with the variable as well?

Reply:
Read the last sentence of my previous comment again <.<